webknossos-datastore/app/com/scalableminds/webknossos/datastore/datareaders/AxisOrder.scala

```diff
@@ -1,8 +1,16 @@
 package com.scalableminds.webknossos.datastore.datareaders
 
-import play.api.libs.json.{Json, OFormat}
+import play.api.libs.json.{JsError, JsNumber, JsObject, JsResult, JsValue, Json, OFormat, Reads, Writes}
+
+sealed abstract class AxisOrder() {
+  def x: Int
+  def y: Int
+  def z: Int
+
+  def c: Option[Int]
+
+  def hasZAxis: Boolean = false
```

Comment:
should not have default value in the abstract class. Also is there a reason not to make it a trait? Is this about the json reads again? :see_no_evil: